CIP-0075 | Fair Stakepool Rewards

TobiasFancee opened the pull request:
New pull request for CIP-0075 because I renamed branch. I also made changes to meet new CIP-0001 requirements.

rphair commented:
Cross-referencing https://github.com/cardano-foundation/CIPs/pull/422#pullrequestreview-1235426080 regarding subjective language in the proposal title.

rphair commented on CIP-Fair_Stakepool_Rewards/README.md:
```diff
@@ -0,0 +1,195 @@
+    - Cardano Foundation
+    - Emurgo
+Discussions:
+    - https://forum.cardano.org/t/cip-fair-stakepool-rewards/109368
```
In my opinion the discussion on the Cardano Forum is fragmented and should also include, if not this link itself (and maybe somewhere except the preamble), then some of the ideas mentioned in this separate discussion thread: https://forum.cardano.org/t/cip-improved-rewards-scheme-parameters/112409

rphair reviewed (commented):
It would be nice to get all the `Discussions:` in as few places as possible, but with this CIP itself being split between this PR and https://github.com/cardano-foundation/CIPs/pull/422 it's perhaps unavailable that the discussion would also be split:

rphair commented on CIP-Fair_Stakepool_Rewards/README.md:
```diff
@@ -0,0 +1,195 @@
+License: CC-BY-4.0
+---
+
+# Abstract
```
```suggestion
# CIP-0013: Cardano URI Scheme

## Abstract
```
... and likewise demoting the headers all the way down to a more web-friendly outline (in which pages only have one H1 and major sections are H2's) ...

rphair commented on CIP-Fair_Stakepool_Rewards/README.md:
```diff
@@ -0,0 +1,195 @@
+
+This proposal retains most of the original reward sharing scheme, but makes changes to ensure fairness, increase decentralization, and reduce the viability of Goldfinger attacks. By removing the minimum fixed fee, adjusting the pledge benefit, increasing k, and reducing a0, a more egalitarian network can be achieved.
+
+# Motivation
+
+## Definitions
```
```suggestion
## Motivation

### Definitions
```
... etc ...

rphair commented on CIP-Fair_Stakepool_Rewards/README.md:
```diff
@@ -0,0 +1,195 @@
+**Sybil Attack**
+- An attack on an online system where an entity tries to take over the network by creating many identities or nodes.
+
+## The Current Rewards Equation
```
```suggestion
### The Current Rewards Equation
```
etc.

rphair reviewed (commented):
@TobiasFancee also just noticed this markdown has the same header rank problem we already fixed for your other CIP in https://github.com/cardano-foundation/CIPs/pull/422#pullrequestreview-1235426080:

prometheus-pool commented:
I think it would be great to mention in this CIP the trade off to removing the unsaturated pledge benefit penalty. Without this part of the reward sharing scheme it seems the protocol would no longer incentivize convergence to k pools since there is no benefit to consolidating stake to k pools if you can get the same rewards at another pool. There are advantages to converging to a specific number of pools such as ensuring that running network infrastructure remains economically viable and that the number of pools doesn't adversely effect network performance. The unsaturated pledge benefit penalty also still results in valuing pledge unlike minPoolCost, so it may perhaps be unfair to lump these both together as anticompetitive.

I think it would also be beneficial to discuss the trade off in reducing a0. It looks like there is currently around 1.3b in private pools earning close to maximum rewards via pledge benefit, but there is around 5x that amount being staked by 0 pledge CEX pools or auto staking wallets like adalite. Private pools make up around 5% of the total stake in the system and their additional rewards come from having skin in the game, while the other entities I've mentioned have 25% of the total stake in the system with 0 skin in the game. I think it is worth pausing to consider that reducing a0 will be giving a raise to these entities with no skin in the game when we actually probably want to give them reason to delegate the ada they control elsewhere, to people that do have skin in the game. Increasing k will also reduce the disparity between those earning maximum rewards now and those delegating to pools with high pledge, helping to mitigate the issue you point out without giving a raise to those with no pledge.

KtorZ commented on CIP-Fair_Stakepool_Rewards/README.md:
```diff
@@ -0,0 +1,195 @@
+Implementors:
+    - Input Output Global
+    - Cardano Foundation
+    - Emurgo
```
Has any of those implementors agreed to be listed here? I hardly think so. As a reminder in CIP-0001:

> Implementors **must sign off on the plan** and be referenced in the document's preamble.

Note also that implementors must be individuals, not entities. Otherwise there's simply no accountability.

rphair replied:
Following on this it's become apparent at the CIP meeting today that this & all other RSS CIP's should only be merged with a status of `Inactive` (also noted in https://github.com/cardano-foundation/CIPs/pull/422#issuecomment-1385152152).

rphair reviewed (commented):
(note at CIP meeting time)

rphair commented on CIP-Fair_Stakepool_Rewards/README.md:
```diff
@@ -0,0 +1,195 @@
+---
+CIP: 75
+Title: Fair Stake Pool Rewards
+Status: Proposed
```
Following on last review it's become apparent at the CIP meeting today that this & all other RSS CIP's should only be merged with a status of `Inactive` (also noted in https://github.com/cardano-foundation/CIPs/pull/422#issuecomment-1385152152).

rphair reviewed (commented):
(note at CIP meeting time)

TobiasFancee commented:
@KtorZ @rphair I've fixed the headers and removed mention of entities with genesis keys. I've set status to "Project Area Not Enlisted for CIP Process" Let me know if this is okay or if "Inactive" would be preferred for status.

rphair commented on CIP-Fair_Stakepool_Rewards/README.md:
```diff
@@ -0,0 +1,193 @@
+---
+CIP: 75
+Title: Fair Stake Pool Rewards
+Status: Project Area Not Enlisted for CIP Process
```
```suggestion
Status: Inactive (project area not enlisted for CIP process)
```
as per https://github.com/cardano-foundation/CIPs/pull/422#discussion_r1123482459

rphair commented on CIP-Fair_Stakepool_Rewards/README.md:
```diff
@@ -0,0 +1,193 @@
+Category: Reward-Sharing Scheme
+Authors:
+    - Tobias Fancee <tobiasfancee@gmail.com>
+Implementors:
+    - TBD
```
```suggestion
Implementors: []
```
as per https://github.com/cardano-foundation/CIPs/pull/422#discussion_r1123471353

rphair commented on CIP-Fair_Stakepool_Rewards/README.md:
```diff
@@ -0,0 +1,193 @@
+License: CC-BY-4.0
+---
+
+# Fair Stake Pool Rewards
+
```
```suggestion
```
as per https://github.com/cardano-foundation/CIPs/pull/422#discussion_r1123467503

rphair commented on CIP-Fair_Stakepool_Rewards/README.md:
```diff
@@ -0,0 +1,193 @@
+
+This proposal retains most of the original reward sharing scheme, but makes changes to ensure fairness, increase decentralization, and reduce the viability of Goldfinger attacks. By removing the minimum fixed fee, adjusting the pledge benefit, increasing k, and reducing a0, a more egalitarian network can be achieved.
+
+## Motivation
```
```suggestion
## Motivation: why is this CIP necessary?
```
as per https://github.com/cardano-foundation/CIPs/pull/422#discussion_r1123472872

rphair commented on CIP-Fair_Stakepool_Rewards/README.md:
```diff
@@ -0,0 +1,193 @@
+| stakePoolTargetNum    | N                   | N                       | 1000           | See Rationale Section.       |
+| poolPledgeInfluence   | N                   | N                       | 0.2            | See Rationale Section.       |
+
+## Rationale
```
```suggestion
## Rationale: how does this CIP achieve its goals?
```
as per https://github.com/cardano-foundation/CIPs/pull/422#discussion_r1123474934

rphair reviewed (commented):
@TobiasFancee this is proceeding in parallel with https://github.com/cardano-foundation/CIPs/pull/422 so likewise I'm ready to approve this after a similar (probably final) round of format changes.

Also likewise, please rename the project folder from `CIP-Fair_Stakepool_Rewards` to `CIP-0075` before we merge it. nerd_face

TobiasFancee commented:
@KtorZ @rphair I've made the requested changes. As with CIP-0082, let me know if there are any more changes needed to merge. Thank you guys for your help throughout this process.

rphair reviewed (approved):
you're very welcome @TobiasFancee and as with parallel https://github.com/cardano-foundation/CIPs/pull/422 I would say this is ready to merge in & around the next CIP meeting (@KtorZ therefore I'm also marking this one `Last Check`).
